tests/system/providers/pinecone/example_pinecone_cohere.py

```diff
@@ -44,8 +43,8 @@ def create_index():
         from airflow.providers.pinecone.hooks.pinecone import PineconeHook
 
         hook = PineconeHook()
-        hook.create_index(index_name=index_name, dimension=768)
-        time.sleep(60)
```

Comment:
Do we need this sleep? Maybe it was added in this task to allow some time to setup the index in backend?

Reply:
Yes, it was just to allow some time to setup the index in backend.

Reply:
Then should we add this sleep back below once after calling create_index because we're removing it and it may cause probs intermittently for the DAG?

Reply:
Added the sleep time as before